apollo-ios/Sources/Apollo/GraphQLQueryWatcher.swift

```diff
@@ -88,6 +88,17 @@ public final class GraphQLQueryWatcher<Query: GraphQLQuery>: Cancellable, Apollo
   public func store(_ store: ApolloStore,
                     didChangeKeys changedKeys: Set<CacheKey>,
                     contextIdentifier: UUID?) {
+    // not implemented, deprecated
```

Comment:
If we move to delegates, I think we'd maintain the existing function. I'm concerned this constitutes a breaking change for consumers of Apollo that have implemented their own `ApolloStoreSubscriber` protocol.